src/Game/AccountManagement.java

```diff
@@ -114,6 +123,222 @@ public static void register(String user, String pass) {
       }
     }
   }
+  
+  public static void forgotPass(String user) {

```

Comment:
Should note above this that this is function is intended to be RSC175 compatible

Reply:
added